tests/unittests/AddressBook/MutableMultiValueTests.mm

```diff
@@ -0,0 +1,154 @@
+//******************************************************************************
+//
+// Copyright (c) 2016 Microsoft Corporation. All rights reserved.
+//
+// This code is licensed under the MIT License (MIT).
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//******************************************************************************
+
+#include <TestFramework.h>
+
+#import <CoreFoundation/CoreFoundation.h>
+#import <Foundation/Foundation.h>
+
+#import "UWP/WindowsApplicationModelContacts.h"
+#import <AddressBook/ABAddressBook.h>
+#import <AddressBook/ABRecord.h>
+#import <AddressBook/ABPerson.h>
+#import <AddressBook/ABMultiValue.h>
+#import <AddressBook/ABMutableMultiValue.h>
+#import "ABContactInternal.h"
+
+void addPhone(WACContact* contact, WACContactPhoneKind kind, NSString* number) {
+    WACContactPhone* phone = [WACContactPhone make];
+    phone.number = number;
+    phone.kind = kind;
+
+    [contact.phones addObject:phone];
+    [phone release];
+}
+
+ABMultiValueRef createMultiValue() {
+    WACContact* contact = [WACContact make];
+    addPhone(contact, WACContactPhoneKindHome, @"0000000000");
+    addPhone(contact, WACContactPhoneKindMobile, @"1111111111");
+    ABRecordRef record = (__bridge_retained ABRecordRef)[[_ABContact alloc] initWithContact:contact];
+    [contact release];
+    ABMultiValueRef multiValue = (ABMultiValueRef)ABRecordCopyValue(record, kABPersonPhoneProperty);
+    CFRelease(record);
+    return multiValue;
+}
+
+TEST(AddressBook, MutableMultiValueCreateTest) {
```

Comment:
>AddressBook,  [](start = 5, length = 13)

nit: with this as the category name, in GTestRunner your tests currently look like:

AddressBook.UnitTests.exe
 |-> AddressBook
    |-> MutableMultiValueCreateTest

With the test class not really doing anything. If you change these to TEST(MutableMultiValue, CreateTest), etc, it'll provide much better filtering.